### DISCUSSION TITLE
Ability To Sort Options of Custom Fields

### DISCUSSION THREAD

**Discussion by chrishildebran** [community/community#11312] [Projects and Issues]
It would be helpful to have the ability of sorting the Options of custom fields. Some of the custom fields I've created have at least a dozen options and the list is growing daily (made easier by being able to create on the fly i must add! Thank you!). 
![2022-02-10_09-24-40](https://user-images.githubusercontent.com/61524205/153462357-44abde00-9339-42b5-ab9d-1285ccf1a8bf.png)

**Comment by Jay-o-Way** [comment]
This is possible
